sorc/build_gfs_util.sh

```diff
@@ -18,4 +18,4 @@ echo ""
 echo " Building ... Executables for GFS_UTILITIES "
 echo ""
 
-source ./compile_gfs_util_wcoss.sh
+source ./compile_gfs_util_${target}.sh
```

Comment:
please revert.

Reply:
Removed compile_gfs_util_jet.sh and reverted build_gfs_util.sh